### ISSUE TITLE
Update R package requirements in "Local Development Configuration" wiki

### ISSUE THREAD

**Issue by ArtemSokolov** [RcppCore/rcpp-gallery#121]
After performing a local build, I noticed that the following R packages were required, but not specified on the ["Local Development Configuration" wiki](https://github.com/RcppCore/rcpp-gallery/wiki/Local-Development-Configuration):

 * RcppXPtrUtils
 * Rcereal
 * nanotime
 * RcppMLPACK
 * [RcppNT2](https://github.com/RcppCore/RcppNT2)
 * [RcppHoney](https://github.com/dcdillon/RcppHoney).

Furthermore, the [benchmark](https://cran.r-project.org/web/packages/benchmark/index.html) package listed in the dependencies seems to be archived on CRAN.

Lastly, I think it might be nice to give wiki readers an `install.packages()` line they can copy to their R environment, similar to what's provided with the `sudo apt-get install ...` lines. Consider including something like
```
install.packages( c("Rcpp", "RcppEigen", ...) )
devtools::install_github( "RcppCore/RcppNT2" )
devtools::install_github( "dcdillon/RcppHoney" )
```

**Comment by eddelbuettel** [comment]
Yes. There is potentially _a lot more_ on a per-post basis, depending on whether you want to recreate the posts.   (I have been noodling with a pdf variant ...)

The `benchmark` is a typo.  I meant [`rbenchmark`](https://cran.r-project.org/package=rbenchmark).

I guess we could clean this up but I am not sure how high a priority it is.  Thoughts, @jjallaire ?

**Comment by jjallaire** [comment]
Not something I have time for right now but definitely worthwhile!

**Comment by eddelbuettel** [comment]
Maybe the simplest fix would be to ... update the wiki page?  

@ArtemSokolov as you have fresh eyes for the topic would you want to take a stab at it?

**Comment by ArtemSokolov** [comment]
Honestly, other than the R package dependencies, I think the wiki page is in a pretty good shape! I'm happy to make the changes I proposed, but I don't think [GitHub has an easy way to submit a pull request for wiki pages](https://stackoverflow.com/questions/10642928/how-to-pull-request-a-wiki-page-on-github).

Let me know how you would like to proceed.

**Comment by eddelbuettel** [comment]
Yes, the wiki pages are often left behind.  I think one ... just edits as on any other wiki but storage is in git so one could roll-up if the need arose.

**Comment by ArtemSokolov** [comment]
I think there's a permission control somewhere. Currently, there is no edit option visible to me in the Wiki tab.

**Comment by eddelbuettel** [comment]
Hm.  Do you see the `Clone this wiki locally` on the right side when you are on a post?

**Comment by ArtemSokolov** [comment]
Yes. However, as discussed in [this blog post](https://www.growingwiththeweb.com/2016/07/enabling-pull-requests-on-github-wikis.html), "while you can clone the wiki and make a change to _your copy_, you cannot submit a pull request as your copy is not “linked” with the original repository."

So, we just need to figure out an effective way to get any changes I make to appear in this primary fork.

**Comment by eddelbuettel** [comment]
Maybe make the changes and email me a diff?  I don't have time to dig now if there is mode somewhere between wiki 'locked and open to owners' and 'open to rest of world'.
